api/SharedWorker.json

```diff
@@ -0,0 +1,241 @@
+{
+  "api": {
+    "SharedWorker": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/SharedWorker",
+        "support": {
+          "webview_android": {
+            "version_added": false
+          },
+          "chrome": {
+            "version_added": "4"
+          },
+          "chrome_android": {
+            "version_added": false
+          },
+          "edge": {
+            "version_added": null
+          },
+          "edge_mobile": {
+            "version_added": null
+          },
+          "firefox": {
+            "version_added": "29"
+          },
+          "firefox_android": {
+            "version_added": "33"
+          },
+          "ie": {
+            "version_added": "10"
+          },
+          "opera": {
+            "version_added": "10.6"
+          },
+          "opera_android": {
+            "version_added": "11.5"
+          },
+          "safari": {
+            "version_added": "5",
+            "version_removed": "6.1"
+          },
+          "safari_ios": {
+            "version_added": "5.1",
+            "version_renoved": "7.1"
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "constructor_name": {
```

Comment:
I think it is the same as under the constructor. Remove this entry.

Reply:
👍